public/app/plugins/datasource/prometheus/components/PromQueryField.tsx

```diff
@@ -333,9 +338,13 @@ class PromQueryField extends React.PureComponent<PromQueryFieldProps, PromQueryF
                   <div className="prom-query-field-info text-warning">
                     {hint.label}{' '}
                     {hint.fix ? (
-                      <a className="text-link muted" onClick={this.onClickHintFix}>
+                      <button
+                        type="button"
+                        className={classNames('text-link', 'muted', clearButtonStyles(theme))}
```

Comment:
i would check what color these buttons are showing as... the `muted` class is trying to set a color, but your `clearButtonStyles` is also setting `color: theme.colors.text.primary`... this might be a case where using `cx` and applying `clearButtonStyles` first makes more sense.